### ISSUE TITLE
TP_DROPS effect deletes inventory from inventory blocks

### ISSUE THREAD

**Issue by bdagenais2** [GC-spigot/AdvancedEnchantments#2639]
### Describe the bug

If you break any inventory block (chest, brewing stand, furnace, etc) that has items in it with a tool enchanted with the "TP_DROPS" effect, you lose the inventory of the items inside that inventory block.

### How to reproduce

Add default enchant "Telepathy" to a pickaxe and break a chest that has diamonds in it.

### "/ae plinfo" link

https://paste.md-5.net/wudenogohe

**Comment by GC-spigot** [comment]
Hi! https://gyazo.com/4b3c5c7301b4f7709ff8d17493c33ea3

I can't seem to recreate this - are you doing this in a protected land or something along those lines?

**Comment by bdagenais2** [comment]
Hello! Thank you for looking into this. It is some kind of incompatibility with "Lands" (https://www.spigotmc.org/resources/lands-land-claim-plugin-grief-prevention-protection-gui-management-nations-wars-1-18-support.53313/).

I haven't figured out why, yet. It is also at least somewhat intermittent. After being AFK for some time, I tested again the issue went away! I reloaded the AdvancedEnchantments plugin and it returned.
